src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,16 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+
+        UserService search = new UserService();
```

Comment:
UserService should be class field
```suggestion
        UserService userServise = new UserService();
```